extern/boostd-data/cmd/run.go

```diff
@@ -233,3 +240,113 @@ var yugabyteMigrateCmd = &cli.Command{
 		return err
 	},
 }
+
+var yugabyteAddIndexCmd = &cli.Command{
+	Name:   "yugabyte-add-index",
+	Usage:  "Add index via boostd-data to YugabyteDB (testing tool)",
+	Before: before,
+	Flags: []cli.Flag{
+		&cli.StringSliceFlag{
+			Name:     "hosts",
+			Usage:    "yugabyte hosts to connect to over cassandra interface eg '127.0.0.1'",
+			Required: true,
+		},
+		&cli.StringFlag{
+			Name:     "connect-string",
+			Usage:    "postgres connect string eg 'postgresql://postgres:postgres@localhost'",
+			Required: true,
+		},
+		&cli.StringFlag{
+			Name:     "filename",
+			Usage:    "filename must be same as pieceCID",
+			Required: true,
+		},
+		&cli.IntFlag{
+			Name:     "CQLTimeout",
```

Comment:
would be good to name the command consistently with the rest i.e. `cql-timeout`

Reply:
agree, but for now i'll stay consistent with the rest of the commands, and every one of the uses `CQLTimeout` at the moment.